event/src/unvalidated/signed/cacao.rs

```diff
@@ -51,20 +48,29 @@ pub struct Payload {
     pub domain: String,
 
     /// Expiration time
+    /// Not using a chrono::DateTime because we need to round trip the exact
+    /// value we receive without modifying precision and changning the CID.
+    /// The new CAIP proposes using ints but most of our cacaos still use ISO 8601 values.
     #[serde(rename = "exp", skip_serializing_if = "Option::is_none")]
-    pub expiration: Option<CapabilityTime>,
+    pub expiration: Option<String>,
```

Comment:
Instead of using a raw String and getting no type safety, would it make sense to introduce a new type, that uses string as the underlying storage but provides a method to convert to a DateTime?

Reply:
Yeah, it might be nice to use a tuple struct with serde transparent. Or at least make these fields private. I can add something like that in the cacao branch I have open.. wanted to fix it before it got released Monday to avoid any weird issues with event serialization.

Reply:
Made the change here: https://github.com/ceramicnetwork/rust-ceramic/pull/502/commits/1d602df9dde4bceaeb25b755d523211822a8b70f